JSDemos/Demos/List/GroupedList/description.md

```diff
@@ -1 +1,11 @@
-The **List** UI component can display items in&nbsp;groups if&nbsp;they are arranged so&nbsp;in&nbsp;the data source. Groups have captions, which are customized in&nbsp;this demo.
\ No newline at end of file
+It is possible to display the **List** in groups under the following conditions:
+
+- The **List** data passed to the **dataSource** is structured so that each group is an object that contains two fields: **key** and **items**. The **key** field represents a group header, and the **items** field represents an array of group items.
```

Comment:
As I mentioned above, I want to use sub-headers to clearly differentiate different methods to supply data. And here I want to have all information regarding data sources that have items already grouped - whether or not field mapping is used. (In your version these things were separated for some reason.)

```suggestion
### Hierarchical Data Source Structure

You can use a **dataSource** where each entry is an object that contains two fields: **key** and **items**. The **key** field specifies the group header (parent), and the **items** field holds an array of child items. This demo shows how to use such a data source.

A data source can specify different field names as long as the data is structured in the same manner. If that's the case, implement the DataSource's [map](/Documentation/ApiReference/Data_Layer/DataSource/Configuration/#map) function to create **key** and **items** field mappings. Refer to the following help topic for more information: [Item Mapping](/Documentation/Guide/Data_Binding/Data_Layer/#Reading_Data/Data_Transformation/Item_Mapping).
```